src/main/php/Handler/LokiHandler.php

```diff
@@ -100,7 +104,7 @@ private function sendPacket(array $packet): void
                     'Content-Type: application/json',
                     'Content-Length: ' . strlen($payload),
                 ],
-            ];
+            ], $this->customCurlOptions);
```

Comment:
Thank you for your changes. there is only one left.
According to PSR-12 this function call should have each argument on a separate line as such :
```php
            $curlOptions = array_replace(
                [
                    CURLOPT_CONNECTTIMEOUT_MS => 100,
                    CURLOPT_TIMEOUT_MS => 200,
                    CURLOPT_CUSTOMREQUEST => 'POST',
                    CURLOPT_RETURNTRANSFER => true,
                    CURLOPT_POSTFIELDS => $payload,
                    CURLOPT_HTTPHEADER => [
                        'Content-Type: application/json',
                        'Content-Length: ' . strlen($payload),
                    ],
                ],
                $this->customCurlOptions
            );
```

Reply:
Reformatted.